.github/workflows/TestMetapackages.yml

```diff
@@ -271,10 +273,12 @@ jobs:
 
       - name: Setup Sass
         run: |
-          ddev yarn set version classic
-          ddev yarn init -y
-          ddev yarn add focus-trap@^6.7.3
-          ddev yarn add file:./drainpipe/metapackages/javascript --dev
+          ddev exec corepack disable
+          ddev exec npm i -g yarn@1.22.1
+          ddev exec 'ln -s $(npm root -g)/yarn/bin/yarn /usr/local/bin/yarn'
+          ddev exec yarn init -y
+          ddev exec yarn add focus-trap@^6.7.3
```

Comment:
what's this package for? Is it just an easy way to do a test?

Reply:
It's testing whether the package import works correctly when bundling